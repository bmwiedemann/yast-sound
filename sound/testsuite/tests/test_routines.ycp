/**
 * File:
 *  test_routines.ycp
 *
 * Module:
 *  Sound
 *
 * Summary:
 *  Tests for sound includes.
 *
 * Authors:
 *  Dan Meszaros <dmeszar@suse.cz>, 2001
 *
 * $Id$
 *
 * Tests for sound routines. 
 */

{
    include "testsuite.ycp";
    include "sound/routines.ycp";

    /*
     *	itemize_list
     */

    list l=["item1", "item2", "item3"];

    TEST (``(is_snd_alias("snd-ward-0")), [], nil);
    TEST (``(is_snd_alias("snd-card0")), [], nil);
    TEST (``(is_snd_alias("snd-rd-0")), [], nil);
    TEST (``(is_snd_alias("snd-card-dummy")), [], nil);
    TEST (``(is_snd_alias("snd-card-1")), [], nil);
    TEST (``(is_snd_alias("snd-card-2")), [], nil);
    

    TEST (``(itemize_list(l, 1)), [], nil);
    TEST (``(itemize_list(l, 4)), [], nil);

}
