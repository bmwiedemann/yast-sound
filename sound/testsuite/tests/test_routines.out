Return	false
Return	false
Return	false
Return	false
Return	true
Return	true
Return	[`item (`id (0), "item1", false), `item (`id (1), "item2", true), `item (`id (2), "item3", false)]
Return	[`item (`id (0), "item1", false), `item (`id (1), "item2", false), `item (`id (2), "item3", false)]
